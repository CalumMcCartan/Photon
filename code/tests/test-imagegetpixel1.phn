func int main()
{
  #whatever
  image img;
  
  int index;
  int p;
  int x;
  int y;
  int position;
  int channels;

  
  img = load("sky.jpg");

  #for gettign a specific pixel, input what you want
  x = 1;
  y = 0;

  #array math, assumes 4 channels
  channels = 4;
  position = (x + (width(img) * y) ) * channels;  

  for( index = position; index < position + 4; index = index+1 ) {
    p = get_pixel(img, index);
    print(p);
    img = load("sky.jpg");
  }

  return 0;
}