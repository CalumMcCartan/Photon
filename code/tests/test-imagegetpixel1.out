139
156
200
255
